profiles/serializers.py

```diff
@@ -41,3 +41,11 @@ class Meta:
         model = SavedCompany
         fields = ('id', 'user', 'company', 'comp_official_name', 'comp_region', 'comp_common_info', 'comp_phone_number',
                   'comp_EDRPOU', 'comp_year_of_foundation', 'comp_address', 'startup_idea', 'added_at')
+
+
+class ViewedCompanySerializer(serializers.ModelSerializer):
+    # profile_id = ProfileSerializer()
+    # user_id =
+    class Meta:
+        model = ViewedCompany
+        fields = "__all__"
```

Comment:
I strongly recommend to not use `__all__`. When you go this way, you can later add some field to the model which you don't actually want to be exposed by API, and it will be exposed.

Reply:
Done.